### PR TITLE
typo (lua_ls)

### PR DESCRIPTION
**Pull request by sahashirshendu**
Sholudn't it be client.config.settings.Lua as that's what is being extended?

### CODE REVIEW

File: doc/server_configurations.txt [hunk 1]
```diff
@@ -4990,7 +4990,7 @@ require'lspconfig'.lua_ls.setup {
   on_init = function(client)
     local path = client.workspace_folders[1].name
     if not vim.loop.fs_stat(path..'/.luarc.json') and not vim.loop.fs_stat(path..'/.luarc.jsonc') then
-      client.config.settings = vim.tbl_deep_extend('force', client.config.settings.Lua, {
+      client.config.settings.Lua = vim.tbl_deep_extend('force', client.config.settings.Lua, {
```
Comment (justinmk) [inline doc/server_configurations.txt:4993]: The automation is asking you to revert this change
```suggestion
      client.config.settings = vim.tbl_deep_extend('force', client.config.settings.Lua, {
```

**Comment by dundargoc** [comment]
@sahashirshendu can you rebase?

**Comment by sahashirshendu** [comment]
> @sahashirshendu can you rebase?

Should I do `git rebase` or `git pull --rebase` in `master` branch?

**Comment by dundargoc** [comment]
I think a `git pull --rebase` or `git rebase` sounds good. I don't trust the Sync fork button lol.

**Comment by sahashirshendu** [comment]
Did that.

**Comment by dundargoc** [comment]
Just create a new PR perhaps?

**Comment by sahashirshendu** [comment]
Fine, will do that. But just being sure, should I edit the `doc/server_configurations.txt` or `lua/lspconfig/server_configurations/lua_ls.lua` or both?
